components/03-modules/dashboard/nav/_nav.scss

```diff
@@ -77,13 +77,19 @@ $dashboard-nav-margin : 15px !default;
 
     // This class is changing by Magento
     .current {
+        position: relative;
+        padding: $dashboard-nav-padding $dashboard-nav-padding $dashboard-nav-padding ($dashboard-nav-padding * 2);
+        margin-left: -$dashboard-nav-margin;
         border-left: 2px solid $red;
         color: $red;
-        position: relative;
 
         &:hover {
             background-color: initial;
         }
+
+        & > a {
```

Comment:
Is this selector necessary ? Can we use some class for example ?

Reply:
it's related to magento theme and links in dashboard, we don't have here any classes on link it's just `a`